packages/abi-gen/diff.sh

```diff
@@ -1,5 +1,5 @@
 diff $1 $2
 if [ $? -ne 0 ]; then
-    echo "ERROR: Freshly generated code does not match known-good, commited output.  If you're confident that the freshly generated code should be considered known-good, then copy it into there and commit it."
+    echo "ERROR: Freshly generated code output does not match expected output.  If you're confident that the expected output should be updated, copy it in there and commit it."
```

Comment:
Nit: remove word "code".  This same file/message should theoretically be able to be copied without change into another package where we want to establish some tests of a command-line interface.